docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc

```diff
@@ -0,0 +1,50 @@
+
+== Storage account is configured with blob anonymous access
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/azure/AzureStorageAccConfigWithoutBlobAnonymousAccess.yaml[CKV2_AZURE_47]
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is identifying whether a storage account has been configured to allow anonymous access to blob data. Anonymous access to blob data means anyone on the internet can retrieve or view data without requiring any form of authentication. This is potentially dangerous because it exposes sensitive data to the public, and it could lead to data leaks and breaches of privacy. Hence, it is critical to ensure storage accounts are not configured with blob anonymous access to safeguard information and maintain data privacy.
+
+=== Fix - Buildtime
+
+*Terraform*
+
+* *Resource:* azurerm_storage_account
+* *Arguments:* allow_nested_items_to_be_public
+
+The problem here is that the Azure storage account is configured with blob anonymous access. Enabling anonymous access to blob content in a storage account allows anyone to read the blob content. This is insecure as sensitive data may be exposed.
```

Comment:
The Azure storage account is configured with anonymous blob access. Enabling anonymous access to blob content in a storage account allows anyone to read the blob content. This is insecure practice, as sensitive data may be exposed.
@tsmithv11 : Why do we need this if there is a description above with the same message?

Reply:
It's redundant. I'll adjust.